### ISSUE TITLE
implicit return-type error during typescript compilation

### ISSUE THREAD

**Issue by KyleCartmell** [redux-observable/redux-observable#96]
When I try to compile my typescript project which uses redux-observable, the following error messages result:

node_modules/redux-observable/index.d.ts(7,3): error TS7010: 'lift', which lacks return-type annotation, implicitly has an 'any' return type.
node_modules/redux-observable/index.d.ts(8,3): error TS7010: 'ofType', which lacks return-type annotation, implicitly has an 'any' return type.

**Comment by KyleCartmell** [comment]
https://github.com/redux-observable/redux-observable/pull/97

**Comment by jayphelps** [comment]
I read #97 and I'm not sure what the status of these two are.

Is there still a problem? could you summarize what the issue is? I see several things mentioned and that PR closed so it's not clear what the issue truly is.

**Comment by KyleCartmell** [comment]
Issue above persists. When I compile a TypeScript project which uses redux-observable tsc complains about implicit 'any' return types. Adding return type information to the typings file exposed by redux-observable resolves the error. The PR I submitted did the needful but used the wrong types. I see your comment in the PR comments stating that ActionsObservable is the correct type, even for ofType. I'll happily open a new PR with the correct types if you feel that this issue is clear and that the types are desirable.
